lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -115,10 +115,13 @@ end |> write_beam
 
 defmodule ExUnit.DocTestTest.NoImport do
   @doc """
-  iex> ExUnit.DocTestTest.NoImport.min(1, 2)
+  iex> ExUnit.DocTestTest.NoImport.max(1, 2)
+  {:ok, 2}
+
+  iex> max(1, 2)
   2
   """
-  def min(a, b), do: max(a, b)
```

Comment:
let's call it `my_min` so we can use `min(a, b)`. This way we don't need to add new lines and not change stacktraces below.